src/kbmod/masking.py

```diff
@@ -2,7 +2,42 @@
 """
 
 import kbmod.search as kb
+from kbmod.search import RawImage
 
+def mask_trajectory(trj, stack, r, bitmask = 1):
+    """ Will apply a circular mask of radius `r` around
+    the trajectory's predicted position in each image.
+
+    Attributes
+    ----------
+    trj : `Trajectory`
+        The trajectory along which to apply the mask.
+    stack : `ImageStack`
+        The stack of images to apply the mask to.
+    r : `float`
+        The radius of the circular mask to apply.
+
+    Returns
+    -------
+    stack : `ImageStack`
+        The stack after the masks have been applied.
+    """
+
+    dt = stack.get_single_image(1).get_obstime() - stack.get_single_image(0).get_obstime()
+    width = stack.get_single_image(0).get_width()
+    height = stack.get_single_image(0).get_height()
+
+    for i in range(stack.img_count()):
+        origin_of_mask = (trj.x + dt * trj.vx * i , trj.y + dt * trj.vy * i) 
+        trajectory_mask = RawImage(width, height)
+
+        for x in range(width):
+            for y in range(height):
```

Comment:
Small comment: Since we store the data row first, this would be a bit more efficient with the `y` loop first.

Reply:
Reordered the loops